tests/saved_test_data/sample_relion_data.star

```diff
@@ -46,4 +46,4 @@ _rlnRandomSubset #29
  2604.810763   203.429484            1     0.729488  -157.620976 000014@sample.mrcs micrograph.mrc   300.000000 21186.804688 21363.109375     7.476096     2.700000     0.000000     1.000000     0.000000     0.100000 37369.207031     5.000000     0.096462          199   118.314252   125.843505    -0.300400    -0.300400     0.699456 1.418293e+05     0.054729           87            1
   757.018043  2660.755621            1     0.932745   -92.812197 000015@sample.mrcs micrograph.mrc   300.000000 21186.804688 21363.109375     7.476096     2.700000     0.000000     1.000000     0.000000     0.100000 37369.207031     5.000000     0.096462          199   134.333245    92.320564    -0.300400     0.199600     0.706685 1.426995e+05     0.074965           42            1
  3165.259343  2881.675032            1     0.513467    97.907345 000016@sample.mrcs micrograph.mrc   300.000000 21186.804688 21363.109375     7.476096     2.700000     0.000000     1.000000     0.000000     0.100000 37369.207031     5.000000     0.096462          199   128.513119    60.205933     0.449600    -0.050400     0.703292 1.428023e+05     0.094331           33            1
-  916.590156  3057.649585            1     0.601333  -120.014857 000017@sample.mrcs micrograph.mrc   300.000000 21186.804688 21363.109375     7.476096     2.700000     0.000000     1.000000     0.000000     0.100000 37369.207031     5.000000     0.096462          199   130.487578    87.229432     0.449600    -0.050400     0.706048 1.425358e+05     0.126520           26            2
```

Comment:
If the only difference between these files is a newline at EOF we should just leave the original.

Unless a dataformat (starfile in this case) says something specific about it, a trailing new line is good practice.